core/src/test/scala/unit/kafka/log/LogCleanerTest.scala

```diff
@@ -137,10 +137,38 @@ class CleanerTest extends JUnitSuite {
                (0 until leo.toInt by 2).forall(!keys.contains(_)))
   }
 
+  def testLogCleanerStats(): Unit = {
```

Comment:
I think it would be nice to have, if not too awkward. If it is too awkward, then OK to skip it since this fixes a bug and improves coverage compared to what we had before.